cmd/traffic_cache_tool/CacheDefs.cc

```diff
@@ -214,7 +214,7 @@ Stripe::Chunk::clear()
   _chain.clear();
 }
 
-Stripe::Stripe(Span *span, Bytes start, CacheStoreBlocks len) : _span(span), _start(start), _len(len)
+Stripe::Stripe(Span *span, Bytes start, CacheStoreBlocks len) : _span(span), _start(start), _len(len), _buckets(0), _segments(0)
```

Comment:
No - put these in the class.

Reply:
why do we want to put these in the class?